script/bump-version.py

```diff
@@ -11,28 +12,76 @@
 
 
 def main():
-  if len(sys.argv) != 2 or sys.argv[1] == '-h':
-    print 'Usage: bump-version.py [<version> | major | minor | patch]'
+
+  parser = argparse.ArgumentParser(
+    description='Bump version numbers. Must specify at least one option.'
+  )
+  parser.add_argument(
+    '--version', 
+    default=None, 
+    dest='new_version', 
+    help='new version number'
+  )
+  parser.add_argument(
+    '--bump', 
+    action='store', 
+    default=None, 
+    dest='bump', 
+    help='increment [major | minor | patch | beta]'
+  )
+  parser.add_argument(
+    '--stable', 
```

Comment:
This assumes that a prerelease exists already. In other words, this won't behave as expected when used for backports, e.g.

```
$ cat package.json | json version 
1.7.6

~/git/electron/electron modify-bump-version
$ npm run bump-version -- --stable --dry-run

> electron@1.7.6 bump-version /Users/z/git/electron/electron
> ./script/bump-version.py "--stable" "--dry-run"

new version number would be: 1.7.6
```

Reply:
Ahh in this case, it would be `npm run bump-version -- --stable --bump=patch --dry-run`; the `bump` argument and `stable` flag aren't mutually exclusive. :) Though I see the documentation is lacking on this front, I will update it with better examples.